db/migrate/20230324074311_addquantityto_food.rb

```diff
@@ -0,0 +1,5 @@
+class AddquantitytoFood < ActiveRecord::Migration[7.0]
+  def change
+    add_column :foods, :quantity, :integer
+  end
+end
```

Comment:
- [ ] Kindly remove this migration file because you already have a quantity column in the foods table. Your app works well without this file.